### PR TITLE
CHANGELOG: Added note on bug fix for bitbucketserver repo description

### PR DESCRIPTION
**Pull request by mibali**
## Test plan
Tested locally 

## App preview:

- [Web](https://sg-web-mibali-bb-server-fix.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,7 @@ All notable changes to Sourcegraph are documented in this file.
 
 - Fixed a bug where saving default Sort & Limit filters in Code Insights did not persist [#46653](https://github.com/sourcegraph/sourcegraph/pull/46653)
 - Restored the old syntax for `repo:contains` filters that was previously removed in version 4.0.0. For now, both the old and new syntaxes are supported to allow for smooth upgrades. Users are encouraged to switch to the new syntax, since the old one may still be removed in a future version.
+- Fixed a bug where the `repo:has.description()` parameter now shows description on Bitbucket server repos, previously it shows the repo name [#46752](https://github.com/sourcegraph/sourcegraph/pull/46752/files)
```
Comment (indradhanush) [inline CHANGELOG.md:38]:
```suggestion
- Fixed a bug where the `repo:has.description()` parameter now shows description on Bitbucket server repos, previously it shows the repo name [#46752](https://github.com/sourcegraph/sourcegraph/pull/46752)
```

**Review by indradhanush — CHANGES_REQUESTED** [review]
Looks good. Requesting changes for the PR title because that goes into the commit message when being merged.

The PR title should reflect what the change is in a short sentence (60-80 chars is a good limit to aim for although it's not super strict).

So in this case something like:

CHANGELOG: Add note on bug fix for bitbucketserver repo description

**Review by BolajiOlajide — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -36,6 +36,7 @@ All notable changes to Sourcegraph are documented in this file.
 - Fixed a bug where saving default Sort & Limit filters in Code Insights did not persist [#46653](https://github.com/sourcegraph/sourcegraph/pull/46653)
 - Restored the old syntax for `repo:contains` filters that was previously removed in version 4.0.0. For now, both the old and new syntaxes are supported to allow for smooth upgrades. Users are encouraged to switch to the new syntax, since the old one may still be removed in a future version.
 - Fixed a bug where removing an auth provider would render a user's Account Security page inaccessible if they still had an external account associated with the removed auth provider. [#47092](https://github.com/sourcegraph/sourcegraph/pull/47092)
+- Fixed a bug where the `repo:has.description()` parameter now shows description on Bitbucket server repos, previously it shows the repo name [#46752](https://github.com/sourcegraph/sourcegraph/pull/46752)
```
Comment (indradhanush) [inline CHANGELOG.md:39]:
```suggestion
- Fixed a bug where the `repo:has.description()` parameter now correctly shows description of a repository synced from a Bitbucket server code host connection, while previously it used to show the repository name instead [#46752](https://github.com/sourcegraph/sourcegraph/pull/46752)
```

**Review by indradhanush — APPROVED** [review]
Left a suggestion. Approved to unblock.

**Pull request merged**